bugbug/tools/code_review.py

```diff
@@ -103,30 +103,21 @@ class LargeDiffError(Exception):
 {comments}"""
 
 
-PROMPT_TEMPLATE_FILTERING_ANALYSIS = """Please, double check the code review provided for the patch below.
-Just report the comments that are:
-- applicable for the patch;
-- consistent with the {target_code_consistency} source code;
-- focusing on reporting possible bugs, major readability regressions, or similar concerns;
-- filter out any descriptive comments;
-- filter out any praising comments.
+PROMPT_TEMPLATE_FILTERING_ANALYSIS = """Filter review comments to identify those that:
+- are consistent with the {target_code_consistency} source code;
+- focus on reporting possible bugs, major functional regressions, major issues, or similar concerns;
+- report major readability or design concerns.
```

Comment:
This is based on https://github.com/mozilla/bugbug/pull/4827#issuecomment-2794557356:

``` suggestion
PROMPT_TEMPLATE_FILTERING_ANALYSIS = """Filter review comments to keep those that:
- are consistent with the {target_code_consistency} source code;
- focus on reporting possible bugs, functional regressions, issues, or similar concerns;
- report readability or design concerns.
```